Cargo.toml

```diff
@@ -8,16 +8,17 @@ keywords = ["hyper", "tls", "http", "https", "ssl"]
 homepage = "https://hyper.rs"
 repository = "https://github.com/hyperium/hyper-tls"
 documentation = "https://docs.rs/hyper-tls"
+edition = "2018"
 
 [features]
 vendored = ["native-tls/vendored"]
 
 [dependencies]
-bytes = "0.4"
-futures = "0.1.21"
+futures-preview = { version = "=0.3.0-alpha.17", features = ["async-await", "nightly"] }
```

Comment:
If we're already using `async`/`await`, then we don't even need this dependency, right? The `future::err` can just be `async { Err(..) }`.

Reply:
You're right. Updated. PTAL.